pages/team.md

```diff
@@ -4,58 +4,73 @@ title: Team
 
 ## Mitzi László
-**Solid Manager** 
-Mitzi graduated in neuroscience from Kings College London before working for a Brazilian epidemiology study funded by the Wellcome Trust specialising in decision making. Her frustration at not being able to access data for public health research is what led her to work on data ethics and security. She has worked as an independent advisor from the European Commission and is a regular public speaker at venues such as the Berkman Klein Centre and the Ada Lovelace Festival as well as featuring on BBC Tomorrow's World. 
+**Solid Manager**
+Mitzi László is Communications Manager at Inrupt. She graduated in neuroscience from Kings College London before working for a Brazilian epidemiology study funded by the Wellcome Trust specialising in decision making. Her frustration at not being able to access data for public health research is what led her to work on data ethics, security, and trust. She has worked as an independent advisor for the European Commission and is a regular public speaker at venues such as the Berkman Klein Centre and the Ada Lovelace Festival as well as featuring on BBC Tomorrow's World.
+
+# Editors
+Editors are responsible for maintaining and reviewing suggestions to the Solid specification.  
 
 ## Sarven Capadisli
-**Solid Editor** 
-Sarven worked on the MIT academic Solid project. He contributes to the advancement of open Web platform specifications and the development of social decentralised applications. He is active as an information architect and scientist in linked research communication.
+**Solid Editor**
+Sarven Capadisli is a technical architect at Inrupt. He contributes to the advancement of open Web platform specifications and the development of social decentralised applications and was part of the MIT Solid project. He is active as an information architect and scientist in linked research communication.
 
-## Ruben Verborgh 
+## Ruben Verborgh
 **Solid Editor**
-Ruben Verborgh is a professor of Decentralized Web technology at IDLab, Ghent University – imec, and a research affiliate at the Decentralized Information Group of CSAIL at MIT. Additionally, he acts as a technology advocate for Inrupt and the Solid ecosystem wherein people and organizations control their own data. He focuses on ecosystem architecture and developer experience.
+Ruben Verborgh is a professor of decentralised Web technology at IDLab, Ghent University – imec, and a research affiliate at the Decentralised Information Group of CSAIL at MIT. Additionally, he acts as a technology advocate for Inrupt and the Solid ecosystem wherein people and organisations control their own data. He focuses on ecosystem architecture and developer experience.
```

Comment:
```suggestion
[Ruben Verborgh](https://ruben.verborgh.org/) focuses on ecosystem architecture and developer experience.
```

With this link, it's perpetually correct and there are no other concerns. We can't rename an MIT group because of a styling guide.